recurring_contract/models/contract_group.py

```diff
@@ -355,6 +395,13 @@ def _process_invoice_generation(self, invoicer, invoicing_date):
             lambda m: getattr(m.invoice_date_due, self.recurring_unit)
             == current_rec_unit_date
         )
+        if len(open_invoice) > 1:
+            _logger.error(
+                f"Found more than one open invoice on {invoicing_date}"
+                f"for the group {self.id}"
+            )
+            return False
```

Comment:
It's maybe a safety check but in that case shouldn't we simply skip to the regular invoice creation rather than aborting completely the invoice generation?
```suggestion
            _logger.error(
                f"Found more than one open invoice on {invoicing_date}"
                f"for the group {self.id}"
            )
            open_invoice = False
```

Reply:
If we are generating invoices for a specific contract it will work since we don't care about other contracts.

But in the case where we are handling a group I think we are going dupplicate invoices, next time it's called tripple it, ect... I wanted to play it safe and stop the execution so that when it occurs, we can investigate it without creating wrong invoices.

Reply:
We decided to raise en error so that we can handle that case whe it comes up.